robosat/losses.py

```diff
@@ -10,17 +11,15 @@ class CrossEntropyLoss2d(nn.Module):
     See: http://cs231n.github.io/neural-networks-2/#losses
     """
 
-    def __init__(self, weight=None, size_average=True, ignore_index=255):
+    def __init__(self, weight=None):
```

Comment:
I think we had the two parameters here to be in line with what the PyTorch losses were doing. I'm fine with removing them.